### PR TITLE
added responsiveness to about page

### LINKED ISSUE #475: About page is not responsive on mobile
## Add relevant screenshot or video 
screenshot

### PR DESCRIPTION
**Pull request by siddhi-244**
### Description
added responsiveness to about page

### Checklist

- [x] I am making a proper pull request, not a spam.
- [x] I've checked the issue list before deciding what to submit.

## Related Issues or Pull Requests
#475 

## Add relevant screenshot or video (if any)

(

)

### CODE REVIEW

File: About/about.css [hunk 1]
```diff
@@ -618,3 +618,17 @@ footer h5 {
...
+
+@media (max-width: 360px){
+ .content{
+  display: block;
+ }
+ .heading{
+  font-size: 20px;
+ }
+}
+@media (max-width: 460px){
```
Comment (niloysikdar) [inline About/about.css]: Change this from 460px to 470px

File: About/about.css [hunk 2]
```diff
@@ -618,3 +618,17 @@ footer h5 {
...
+ .heading{
+  font-size: 20px;
+ }
```
Comment (niloysikdar) [inline About/about.css:633]: Add another CSS property here.
```css
.content {
    overflow: hidden;
}
```

**Review by niloysikdar — CHANGES_REQUESTED** [review]
Please make the requested changes @siddhi-244

**Comment by siddhi-244** [comment]
@niloysikdar now please see !

**Review by niloysikdar — APPROVED** [review]
Thanks @siddhi-244

**Pull request merged**